PostBoard.Api/Controllers/PingController.cs

```diff
@@ -9,6 +9,6 @@ public class PingController : ControllerBase
     [HttpGet]
     public IActionResult Ping()
     {
-        return Ok();
+        return  Ok();
```

Comment:
Remove unnecessary space before `Ok(..)`.

Reply:
This method should be asyncronous, but without any `await` inside. So you can return update this method like this:
```
public Task<IActionResult> PingAsync()
{
    IActionResult response = Ok();
    return Task.FromResult(response);
}
```